lms/djangoapps/ccx/api/v0/tests/test_views.py

```diff
@@ -872,7 +872,7 @@ def test_get_detail(self):
         )
         self.assertEqual(resp.data.get('coach_email'), self.ccx.coach.email)  # pylint: disable=no-member
         self.assertEqual(resp.data.get('master_course_id'), unicode(self.ccx.course_id))  # pylint: disable=no-member
-        self.assertEqual(resp.data.get('course_modules'), self.master_course_chapters)  # pylint: disable=no-member
+        self.assertItemsEqual(resp.data.get('course_modules'), self.master_course_chapters)  # pylint: disable=no-member
```

Comment:
Is the pylint pragma complaining about `data` not being a member on `resp`?  If so, it might be worth assigning `data = resp.data` above, so the pragma can be used just once. (Nit)